### ISSUE TITLE
Request - Account unlock denied - Help

### ISSUE THREAD

**Issue by Kmullineaux** [ArctosDB/arctos#8999]
Help us understand your request (check below):

- [x] update to an Agent (split, merge, or other type of change)
- [ ] error or bug (please include enough specific information to reproduce)
**Describe what you're trying to do** 

I am attempting to unlock the account of my volunteer who wasn't working in Arctos for a while but now needs access again. Her username is rbaldwin, and this is the error code: 

<html>
<body>

ERROR_ID | A92E3DBD-4D1C-4C8A-8A11CC639A8901F7
-- | --
ERROR_TYPE | application
ERROR_MESSAGE | unlock denied
ERROR_DETAIL | Local-authentication Operator accounts may not be unlocked here; contact the admin team with Arctos and TACC usernames for assistance

</body>
</html>

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by dustymc** [comment]
You should be able to unlock rbaldwin now, and they can log into Arctos using TACC. If you meant their TACC account isn't set up for MFA they'll have to do that, https://handbook.arctosdb.org/documentation/oauth.html should help.
